frontend/package-lock.json

```diff
@@ -167,7 +167,8 @@
       "version": "1.0.0",
```

Comment:
I'm not sure about these changes to `package-lock.json`. Why are they suddenly optional? This does not happen on my machine. You work on a Mac, right? I'd like to investigate further, before accepting these. :thinking:

Reply:
Yes, I'm work in a mac, maybe that is why created that.

Reply:
So can you remove these changes for now? :wink:

Reply:
Done! 😄